auacm/app/modules/submission_manager/views.py

```diff
@@ -48,191 +28,23 @@ def submit():
     uploaded_file = request.files['file']
     if not uploaded_file:
         return serve_error('file must be uploaded', response_code=400)
-    if not allowed_filetype(uploaded_file.filename):
+    if not judge.allowed_filetype(uploaded_file.filename):
         return serve_error('filename not allowed', response_code=403)
     if not request.form['pid']:
         return serve_error('the field \'pid\' must be specified', response_code=400)
 
-    attempt = Submission(username=current_user.username,
-                         pid=request.form['pid'],
-                         submit_time=int(time.time()),
-                         auto_id=0,
-                         file_type=uploaded_file.filename.rsplit('.', 1)[1].lower(),
-                         result='start')
-    session.add(attempt)
-    session.flush()
-    session.commit()
-    session.refresh(attempt)
-    directory = directory_for_submission(attempt)
-    os.mkdir(directory)
-    uploaded_file.save(join(directory, uploaded_file.filename))
-    thread = Thread(target=start_execution, args=(attempt, uploaded_file))
+    attempt = models.Submission(
```

Comment:
It's cool if you want to refactor the code, (and I like this style better than the one that I initially used) but in the future, let's avoid changing style during commits. It's not truly productive and can end being a battle.